module12/src/main/java/com/TEKcamp/module12/Controllers/UserController.java

```diff
@@ -0,0 +1,68 @@
+package com.TEKcamp.module12.Controllers;
+
+import com.TEKcamp.module12.Entities.User;
+import com.TEKcamp.module12.Repositories.UserRepository;
+import com.TEKcamp.module12.Services.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController //convenience annotation instead of @Controller, includes ResponseBody
+@RequestMapping("users") //where it's listening
+
+public class UserController {
+    private final UserService userService;
+    private final UserRepository userRepository;
+
+    public UserController(UserService userService, UserRepository userRepository) {
+        this.userService = userService;
+        this.userRepository = userRepository;
+    }
+
+    public UserService getUserService() {
+        return userService;
+    }
+
+    public UserRepository getUserRepository() {
+        return userRepository;
```

Comment:
You don't need getters for these. You're only going to use these instances inside this class, which does not require getters